extensions/notebook/src/jupyter/jupyterServerManager.ts

```diff
@@ -55,15 +55,33 @@ export class LocalJupyterServerManager implements nb.ServerManager, vscode.Dispo
 		try {
 			if (!this._jupyterServer) {
 				this._jupyterServer = await this.doStartServer(kernelSpec);
+
+				// Ensure the server is ready before firing the event
+				await this.verifyServerReady(this._jupyterServer.uri.toString());
```

Comment:
Do we still need this verification step if we await the server `start` call [here](https://github.com/microsoft/azuredatastudio/blob/main/extensions/notebook/src/jupyter/jupyterServerManager.ts#L149)?

Reply:
there is already an await there, so yes, this step is needed for the notebook to work properly.